permissions_validators/src/private_blockchain/query.rs

```diff
@@ -326,9 +336,9 @@ impl<W: WorldTrait> IsAllowed<W, QueryBox> for OnlyAccountsData {
                     .id
                     .evaluate(wsv, &context)
                     .map_err(|e| e.to_string())?;
-                if let Ok(true) = wsv.world.triggers.inspect(&id, |action|
+                if wsv.world.triggers.inspect(&id, |action|
                     action.technical_account() == authority
-                ) {
+                ) == Some(true) {
```

Comment:
I understand, what and why you are talking about, but I think it's more relevant to other languages rather than Rust. Also this is actually a clippy [suggestion](https://rust-lang.github.io/rust-clippy/master/index.html#equatable_if_let) and it warns if I put the constant first

Reply:
Good to know.